core-container/entry.sh

```diff
@@ -23,17 +23,7 @@ function echoerr() {
    echo "ERROR: $1" >&2
 }
 
-
-function set_provider_adapters() {
-    oldKubeconfig=$KUBECONFIG
-    KUBECONFIG=
-    echo "Waiting for space manager to be ready... this may take a while."
-    (
-        until [ "$(kubectl logs $(kubectl get pod --selector=app=kubestellar -o jsonpath='{.items[0].metadata.name}') -c space-manager | grep '***READY***')" != "" ]; do
-            sleep 10
-        done
-    )
-
+function create_kcp_provider() {
     echo "Delete the kcp provider secret if it already exists, and then create it."
     if ! kubectl delete secret -n ${NAMESPACE} kcpsec ; then
         echo "Nothing to delete."
```

Comment:
This message omits lots information, I think it will leave readers wondering: what is it talking about deleting, and why?